src/table/hooks/use-column-widths.ts

```diff
@@ -27,7 +27,7 @@ type GetFitToContentWidth = (headLabel: string, totalsLabel: string, glyphCount:
  * The widths are sorted in the order they will be displayed
  */
 export const getColumnWidths = (columns: Column[], tableWidth: number, getFitToContentWidth: GetFitToContentWidth) => {
-  if (!columns?.length || tableWidth === 0) return [];
+  if (!columns?.length) return [];
```

Comment:
Still keeping this sanity check. Not sure if it is needed, but not causing any issues either